src/NumericalAlgorithms/FiniteDifference/DerivativeOrder.hpp

```diff
@@ -0,0 +1,53 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <iosfwd>
+
+/// \cond
+namespace Options {
+class Option;
+template <typename T>
+struct create_from_yaml;
+}  // namespace Options
+/// \endcond
+
+namespace fd {
+/// Controls which FD derivative order is used.
+enum class DerivativeOrder : int {
+  /// \brief Use one order high derivative.
+  ///
+  /// For example, if fifth order reconstruction is used, then a sixth-order
+  /// derivative is used.
+  OneHigherThanRecons = -1,
+  /// \brief Same as DerOneHigher except uses a fourth-order derivative if
```

Comment:
`Der`?

Reply:
Whoops... Should be `OneHigherThanRecons`... Squashed this in. Thanks for the review!